site/building.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Developing and Debugging
+menu_order: 90
+---
+
+The following topics are discussed: 
+
+ * [Developing](#developing)
+ * [Debugging](#debugging)
+
+
+## <a name="developing"></a>Developing
+
+Building Scope from source depends on the latest version of [docker](https://www.docker.com/), so please install that before proceeding.
+
+The main build is `make`, which builds the UI build container, builds the UI in said container, builds the backend build container, builds the app and probe in said container, and finally pushes the lot into a Docker image called **weaveworks/scope**.
```

Comment:
> The main build is `make`

This reads weirdly.  The main build tool is make?  The build is orchestrated by make?  The build process is automated using make?